openhands/llm/llm.py

```diff
@@ -159,11 +159,12 @@ def __init__(
                     self.config.max_output_tokens = self.model_info['max_tokens']
 
         self.config.supports_function_calling = (
-            self.model_info is not None
-            and self.model_info.get('supports_function_calling', False)
+            (self.model_info is not None and self.model_info.get('supports_function_calling', False))
+            or 'deepseek' in self.config.model.lower()  # DeepSeek models support function calling
```

Comment:
You don't need to check if DeepSeek supports function calling. Let's assume model_info already returns True (it does).